overlord/overlord.go

```diff
@@ -80,6 +80,7 @@ type Overlord struct {
 	deviceMgr  *devicestate.DeviceManager
 	cmdMgr     *cmdstate.CommandManager
 	unknownMgr *UnknownTaskManager
+	udevMon    udevMonitor
```

Comment:
Nitpick: hardwareManager

Reply:
Created HardwareManager and moved UDevMonitor into it.